config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -120,126 +120,6 @@ presubmits:
           defaultMode: 256
           secretName: ssh-security
     trigger: (?m)^/test( | .* )pull-security-kubernetes-e2e-gce,?($|\s.*)
-  - agent: kubernetes
```

Comment:
/shrug

it's just running `/hack/update-config`, and reordering the security jobs